src/ontology/cl-edit.owl

```diff
@@ -32081,6 +32083,27 @@ AnnotationAssertion(rdfs:label obo:CL_4042011 "subpial interlaminar astrocyte"@e
 SubClassOf(obo:CL_4042011 obo:CL_4042009)
 SubClassOf(obo:CL_4042011 ObjectSomeValuesFrom(obo:RO_0002170 obo:UBERON_0002361))
 
+# Class: obo:CL_4042012 (L2 marginal neuron)
+
+AnnotationAssertion(obo:IAO_0000028 obo:CL_4042012 "L2MN")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:29311847") Annotation(oboInOwl:hasDbXref "PMID:32108571") obo:IAO_0000115 obo:CL_4042012 "An L2 intratelencephalic projecting glutamatergic neuron with a soma on the L1-L2 border. This neuron type has small apical dendrites projecting to L1.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4042012 <https://orcid.org/0000-0002-0098-8958>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4042012 "2024-04-23T09:31:39Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:29311847") Annotation(oboInOwl:hasDbXref "PMID:32108571") oboInOwl:hasExactSynonym obo:CL_4042012 "L2MN")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32108571") oboInOwl:hasExactSynonym obo:CL_4042012 "layer 2 marginal neuron")
+AnnotationAssertion(rdfs:label obo:CL_4042012 "L2 marginal neuron"@en)
+SubClassOf(obo:CL_4042012 obo:CL_4023040)
```

Comment:
I think you were right about [L2 intratelencephalic projecting glutamatergic neuron](http://purl.obolibrary.org/obo/CL_4030060), my concerns were not on the layer, but on the intratelencephalic projecting glutamatergic neuron, as I had to do more reading.